devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy

```diff
@@ -0,0 +1,225 @@
+								[value: 74, color: "#44b621"],
+								[value: 84, color: "#f1d801"],
+								[value: 95, color: "#d04e00"],
+								[value: 96, color: "#bc2323"]
+						]
+			}
+		}
+		valueTile("humidity", "device.humidity", inactiveLabel: false, width: 2, height: 2) {
+			state "humidity", label: '${currentValue}% humidity', unit: ""
+		}
+		valueTile("dewPoint", "device.dewPoint", inactiveLabel: false, width: 2, height: 2) {
+			state "dewPoint", label: '${currentValue}° dewPoint', unit: ""
+		}
+		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label: "", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "temperature", "humidity", "dewPoint"
+		details(["temperature", "humidity", "dewPoint", "battery"])
+	}
+}
+
+def updated() {
+	configure()
+}
+
+def installed() {
+	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 10 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+
+	def cmds = [
+		secure(zwave.wakeUpV2.wakeUpIntervalSet(seconds: 4 * 3600, nodeid: zwaveHubNodeId)),
+		secure(zwave.wakeUpV2.wakeUpIntervalGet())
+	]
+
+	response(cmds)
+}
+
+def configure() {
+	delayBetween([
+			secure(zwave.batteryV1.batteryGet()),
+			secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x05)), // humidity
+			secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x01)), // temperature
+			secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x0B)), // dew point
+			secure(zwave.notificationV3.notificationGet(notificationType: 0x10)), // mold detection
+			secure(zwave.wakeUpV2.wakeUpNoMoreInformation())
+	], 2000)
+}
+
+def parse(String description) {
+	def results = []
+
+	if (description.startsWith("Err")) {
+		results += createEvent(descriptionText: description, displayed: true)
+	} else {
+		def cmd = zwave.parse(description)
+		if (cmd) {
+			results += zwaveEvent(cmd)
+		}
+	}
+
+	log.debug "parse() result ${results.inspect()}"
+
+	return results
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.wakeupv2.WakeUpIntervalReport cmd) {
+	log.debug "Wake Up Interval Report: ${cmd}"
+}
+
+def getRefreshCommands() {
+	def cmds = [secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x05)).format(), // humidity
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x01)).format(), // temperature
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x0B)).format(), // dew point
+		secure(zwave.notificationV3.notificationGet(notificationType: 0x10)).format()] // mold detection
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd) {
+	log.debug "Event: ${cmd.event}, Notification type: ${cmd.notificationType}"
+
+	getRefreshCommands()
```

Comment:
These will not be sent to the device. You'd need to wrap in `sendHubCommand` i think